Tests/MongoSwiftTests/MongoCollection+IndexTests.swift

```diff
@@ -2,6 +2,8 @@
 import Nimble
 import XCTest
 
+private var _client: MongoClient?
```

Comment:
we were inadvertently using the shared client defined in `MongoCollectionTests.swift` which seemed a little weird